armi/reactor/converters/axialExpansionChanger.py

```diff
@@ -257,9 +259,10 @@ def manageCoreMesh(self, r):
 
         oldMesh = r.core.p.axialMesh
         r.core.updateAxialMesh()
-        runLog.extra("Updated r.core.p.axialMesh (old, new)")
-        for old, new in zip(oldMesh, r.core.p.axialMesh):
-            runLog.extra(f"{old:.6e}\t{new:.6e}")
+        if oldMesh:
+            runLog.extra("Updated r.core.p.axialMesh (old, new)")
```

Comment:
Let's format this better. The output is pretty ugly. You could use tabulate.

Reply:
This is a good idea. Just going to (temporarily) punt to an issue to keep the content of this PR as minimal as possible.

Reply:
Issue here https://github.com/terrapower/armi/issues/833